src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,81 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMA = ",";
+    private static final String NOTWORD = "\\W+";
```

Comment:
```suggestion
    private static final String NOT_WORD = "\\W+";
```

Reply:
You need to follow the naming convention. In Java, constants are typically named using all capital letters and underscores to separate words, if the constant consists of multiple words.

Reply:
For this one you can use also such names as `REGEX_DELIMITER` or `WORD_REGEX_DELIMITER`.